php/libraries/LorisForm.class.inc

```diff
@@ -304,6 +324,9 @@ class LorisForm
         case 'header':
             $el = $this->addHeader($name, $label);
             break;
+        case 'advcheckbox':
```

Comment:
advcheckbox is what QuickForm called it. A HTML checkbox either submits "true" or doesn't include anything at all. I think the idea behind an advcheckbox is that it either submits "true" or "false", not "true" or nothing.